### ISSUE TITLE
Sprint 2 Review

### ISSUE THREAD

**Issue by nep2ne** [are-you-still-watching/web-app#17]
We completed the UX and visual design, including creating a logo and name for our app, we created sitemaps/flow chart, persona, rough wireframes and high-fidelity wireframes for the Home, Profile and Details pages. We conducted background research and a competitive analysis review. 
@thomasjbradley

**Comment by nep2ne** [comment]
The sprint succeeded in that we all collaborated on the design and background UX document. We each played to our strengths and helped each other complete the sprint issues. I took the lead on writing the background document and completing the competitive analysis, and I created rough sketches / wireframes, a sitemap/flowchart, and I also created the rubrics for the UX and the code. 

The sprint didn't really fail in any way. We finished all the tasks that we had assigned each other, the workload distribution seemed fair, and the quality of our work was high.

**Comment by KyleVBP** [comment]
Our sprint succeeded really well, all team members were on track and completed their parts quickly and on time. We all brought our own style to the final UX.

The sprint didn't have much of any failure. Everyone had their own work and all tasks were completed at a high quality level.

**Comment by moutonrebelle** [comment]
**How did you & the sprint succeed? What was positive?**
We successfully organized and assigned tasks to each member of the team. My task was to create a logo and a high-fidelity wireframe for the "profile" section. The work distribution seemed fair for everybody and the quality of our work was well executed.

The sprint was successful and didn't really have any failure since we all worked together efficiently.

**Comment by Sebby-T** [comment]
**How did you & the sprint succeed? What was Positive?**
We organized and delegated all tasks to the appropriate team members. Everyone work to their strengths and each task was manageable within the given time frame. Everyone shared and completed their part and to my knowledge, no one felt overburdened. I personally handled the wireframes and visual design along with providing input to teammates work and helping solve issues. We worked great as a team.

**How did you & the sprint fail? What was negative?**
There was no sign of failing any of the sprint's tasks. Everyone completed on time.
